app.js

```diff
@@ -139,4 +150,108 @@ async function getAgendaitemsRequestUrls(agendaId) {
   return urls;
 }
 
+async function warmupConcepts() {
+  const urls = [
+    await getConceptsHasNoNarrowerBatchedRequestsUrls(CONCEPT_SCHEMES.MEETING_TYPE),
+    await Promise.all([
+      CONCEPT_SCHEMES.AGENDA_ITEM_TYPES,
+      CONCEPT_SCHEMES.MEETING_TYPE,
+      CONCEPT_SCHEMES.ACCESS_LEVELS,
+      CONCEPT_SCHEMES.DOCUMENT_TYPES,
+      CONCEPT_SCHEMES.DECISION_RESULT_CODES,
+      CONCEPT_SCHEMES.RELEASE_STATUSES,
+      CONCEPT_SCHEMES.USER_ROLES,
+      CONCEPT_SCHEMES.GOVERNMENT_FIELDS,
+    ].map(getConceptsBatchedRequestsUrls)),
+    STATIC_TYPES.map(getStaticTypeUrl),
+  ].flat(Infinity);
+
+  for (let group of MU_AUTH_ALLOWED_GROUPS) {
+    const allowedGroupHeader = JSON.stringify(group);
+    console.log(
+      `Warming up all concepts for allowed group ${allowedGroupHeader} (${urls.length} requests, executed in parallel)`
+    );
+    const promises = urls.map((url) => {
+      return fetch(url, {
+        method: "GET",
+        headers: {
+          "mu-auth-allowed-groups": allowedGroupHeader,
+        },
+      })
+    });
+
+    await Promise.all(promises);
+  }
+}
+
+async function getConceptsHasNoNarrowerBatchedRequestsUrls(conceptSchemeUri) {
+  const urls = [];
+
+  // count
+  const countParams = new URLSearchParams({
+    "filter[:has-no:narrower]": true,
+    "filter[concept-schemes][:uri:]": conceptSchemeUri,
+    include: "broader,narrower",
+    "page[size]": 1,
+    sort: "position",
+  });
+  urls.push(`${BACKEND_URL}concepts?${countParams}`);
+
+  const count = await helpers.countConceptsForConceptScheme(conceptSchemeUri);
+
+  // the batches
+  const batchSize = 100;
+  const nbOfBatches = Math.ceil(count / batchSize);
+  for (let i = 0; i < nbOfBatches; i++) {
+    const params = new URLSearchParams({
+      "filter[:has-no:narrower]": true,
+      "filter[concept-schemes][:uri:]": conceptSchemeUri,
+      include: "broader,narrower",
```

Comment:
See comment in FE as well, including `narrower` while filtering on `:has-no:narrower` looks a bit silly